lib/collections/schemas/helpers.js

```diff
@@ -99,3 +99,82 @@ export function shopDefaultCountry() {
     return this.value;
   }
 }
+
+/**
+ * shopDefaultParcelSize
+ * @namespace shopDefaultParcelSize
+ * @memberof schemas
+ * @method
+ * @summary Helper method used in schema injection autoValue for product variant weight, length, width, and height
+ * @example autoValue: weight
+ * @return {String} parcel weight value from shop default parcel size
+*/
+export const shopDefaultParcelSize = {
+
+  /**
+   * A function in shopDefaultParcelSize (shopDefaultParcelSize.currentShop).
+   * @function currentShop
+   * @memberof shopDefaultParcelSize
+   * @summary Function to get current shop from database
+  */
+  currentShop: function () {
+    return Shops.findOne({
+      _id: Reaction.getShopId()
+    });
+  },
+
+  /**
+   * A function in shopDefaultParcelSize (shopDefaultParcelSize.weight).
+   * @function weight
+   * @memberof shopDefaultParcelSize
+   * @summary Function to get weight value from shop default parcel size
+  */
+  weight: function () {
+    const shop = new shopDefaultParcelSize.currentShop();
+    if (shop && shop.defaultParcelSize) {
```

Comment:
@zenweasel could this be `false` anywhere?

Reply:
@Akarshit yes. It could be false if defaultParcelSize is not defined in [Shops.json](https://github.com/reactioncommerce/reaction/blob/efb71a757a9a2c5058af3b55dcedb65854b4f786/private/data/Shops.json). After a discussion with @zenweasel we decided to also set autoValues for defaultParcelSize in [shops.js](https://github.com/reactioncommerce/reaction/blob/efb71a757a9a2c5058af3b55dcedb65854b4f786/lib/collections/schemas/shops.js).